src/plugins/dna_stat/src/DNAStatMSAProfileDialog.h

```diff
@@ -43,7 +43,7 @@ class DNAStatMSAProfileDialog : public QDialog, public Ui_DNAStatMSAProfileDialo
     DNAStatMSAProfileDialog(QWidget *p, MSAEditor *ctx);
 
     void accept();
-
+    void alignmentIsTooBig();
```

Comment:
Start method names with a verb: rename to showAlignmentIsTooBigWarning() or switchIntoBigAlignmentMode()